components/layout/InvestigationHero/styles.js

```diff
@@ -56,21 +67,73 @@ export const Text = styled.div`
   flex-direction: column;
   justify-content: center;
   align-items: flex-start;
+
+  ${respond(
+    `
+    margin-right: 60px;
+    margin-left: 60px;
+    `,
+    tokens.BREAK_TABLET
+  )}
+
+  ${respond(
+    `
+    margin-right: 15px;
+    margin-left: 15px;
+    `,
+    tokens.BREAK_MOBILE
+  )}
 `;
 
 export const ButtonWrapper = styled.div`
   grid-area: button;
 
   ${respond(
-    css`
+    `
       width: 100%;
       display: flex;
       flex-direction: column;
+      padding-inline-start: ${BUTTON_WRAPPER_PADDING};
+      padding-inline-end: ${BUTTON_WRAPPER_PADDING};
     `,
     tokens.BREAK_TABLET
   )}
 `;
 
+export const EarlyAccessFlag = styled(EarlyAccess)`
+  position: absolute;
+  top: 0;
+  right: ${DURATION_WIDTH};
+
+  @media (max-width: ${tokens.BREAK_TABLET}) {
+    right: auto;
+    left: 0;
+  }
+
+  ${OffsetWrapper} {
+    margin-right: -100%;
```

Comment:
I think this is a legit "css trick" kinda thing, but I didn't really test across browsers. My "offset an element so that it can be positioned along its center axis" skills are a bit rusty. Better more bulletproof way to accomplish this?

Reply:
I think using 

```css
    translateX(-100%);
```

would be the cleanest way to accomplish this